src/test/java/test/utils/domain/NoIdEntity.java

```diff
@@ -0,0 +1,34 @@
+package test.utils.domain;
+
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.keyvalue.annotation.KeySpace;
+
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * An entity with no {@link Id} annotation
+ *
+ * @author Gokhan Oner
+ */
+@KeySpace
+public class NoIdEntity implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    private String name;
+
+    public String getName() {
```

Comment:
Do you need this method for test? If not, I'd remove it to keep this class as small as possible. The same with `setName()` and `toString()`.